src/Domain/CoordinateMaps/CoordinateMap.tpp

```
@@ -0,0 +1,417 @@
+                    map_in_loop)>{});
+            jac_overload(&noframe_jac, map, mapped_point, time,
+                         functions_of_time,
+                         CoordinateMap_detail::is_jacobian_time_dependent_t<
+                             decltype(map), T>{});
+            std::array<T, dim> temp{};
+            for (size_t source = 0; source < dim; ++source) {
+              for (size_t target = 0; target < dim; ++target) {
+                gsl::at(temp, target) =
+                    noframe_jac.get(target, 0) * jac.get(0, source);
+                for (size_t dummy = 1; dummy < dim; ++dummy) {
+                  gsl::at(temp, target) +=
+                      noframe_jac.get(target, dummy) * jac.get(dummy, source);
+                }
+              }
+              for (size_t target = 0; target < dim; ++target) {
+                jac.get(target, source) = std::move(gsl::at(temp, target));
+              }
+            }
+          }
+        } else {
+          jac_overload(
+              &noframe_jac, map, mapped_point, time, functions_of_time,
+              CoordinateMap_detail::is_jacobian_time_dependent_t<decltype(map),
+                                                                 T>{});
+          for (size_t target = 0; target < dim; ++target) {
+            for (size_t source = 0; source < dim; ++source) {
+              jac.get(target, source) =
+                  std::move(noframe_jac.get(target, source));
+            }
+          }
+        }
+      },
+      maps_);
+  return jac;
+}
+
+template <typename SourceFrame, typename TargetFrame, typename... Maps>
+bool operator!=(
+    const CoordinateMap<SourceFrame, TargetFrame, Maps...>& lhs,
+    const CoordinateMap<SourceFrame, TargetFrame, Maps...>& rhs) noexcept {
+  return not(lhs == rhs);
+}
+
+template <typename SourceFrame, typename TargetFrame, typename... Maps>
+auto make_coordinate_map(Maps&&... maps)
```

function is not labelled `noexcept`. Is there a reason to let it throw?

---

No, I just didn't change it to make this as much of a straight copy-paste as possible. I'm happy to add it if you would like, either now or in a separate PR or commit. Whatever you prefer :)